faculty/clients/experiment.py

```diff
@@ -293,3 +333,34 @@ def list_runs(
         return self._get(
             endpoint, ListExperimentRunsResponseSchema(), params=query_params
         )
+
+    def log_run_data(
+        self, project_id, run_id, metrics=None, params=None, tags=None
+    ):
+        """Update the data of a run.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
+        run_id : uuid.UUID
+        metrics : List[experiments.ExperimentRunMetric], optional
+            Each metric will be inserted.
+        params : List[experiments.ExperimentRunMetric], optional
+            Each param will be inserted and rejected if the given key is
+            present.
+        tags : List[experiments.ExperimentRunTag], optional
+            Each tag be upserted.
+
+        Returns
+        -------
+        None
+        """
+        endpoint = "/project/{}/run/{}/data".format(project_id, run_id)
+        payload = ExperimentRunDataSchema().dump(
+            {
+                "metrics": ExperimentRunMetricSchema(many=True).dump(metrics),
```

Comment:
The `ExperimentRunDataSchema` should use the `ExperimetnRunMetricSchema` etc. internally by itself - why have you pre-serialised the data here before passing it to `ExperimentRunDataSchema.dump` ?

Reply:
I thought I needed to. Changed